java/java.j2seplatform/manifest.mf

```diff
@@ -4,5 +4,5 @@ OpenIDE-Module-Localizing-Bundle: org/netbeans/modules/java/j2seplatform/Bundle.
 OpenIDE-Module-Layer: org/netbeans/modules/java/j2seplatform/resources/layer.xml
 OpenIDE-Module-Install: org/netbeans/modules/java/j2seplatform/J2SEPlatformModule.class
 AutoUpdate-Show-In-Client: false
-OpenIDE-Module-Specification-Version: 1.49
+OpenIDE-Module-Specification-Version: 1.50
```

Comment:
I'm unsure as to when this needs to be incremented. Otherwise looks OK.

Reply:
@JaroslavTulach or @jlahoda can we have a confirmation for the increment ?